core/domain/skill_services.py

```diff
@@ -397,6 +394,7 @@ def _save_skill(committer_id, skill, commit_message, change_list):
     skill_model.misconceptions = [
         misconception.to_dict() for misconception in skill.misconceptions
     ]
+    change_list = [change.to_dict() for change in change_list]
```

Comment:
Don't repurpose vars (cf comments in other PRs)

Reply:
Done.